dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/MongoDBQueryEngine.java

```diff
@@ -118,7 +124,8 @@ public MongoDBRdfConfiguration getConf() {
             }
 
             // TODO not sure what to do about regex ranges?
-            final RyaStatementBindingSetCursorIterator iterator = new RyaStatementBindingSetCursorIterator(coll, rangeMap, strategy, conf.getAuthorizations());
+            final RyaStatementBindingSetCursorIterator iterator = new RyaStatementBindingSetCursorIterator(
+                    getCollection(conf), rangeMap, strategy, conf.getAuthorizations());
```

Comment:
Since you call getCollection(conf), I believe that coll is now an unused field.  Can you remove it?